source/guides/exchanges/exchange-REST-troubleshooting.rst

```diff
@@ -0,0 +1,218 @@
+
+The following sections aim at clarifying the points which have been deemed the most confusing by users of the API.
+
+Transaction deadline
+====================
+
+Transactions are not allowed to remain unconfirmed in the network forever, as this would pose a significant strain on the network's resources. Instead, **all transactions have a deadline**, and are automatically disposed of when the deadline arrives.
+
+Users are free to use any deadline they want for their transactions, between **now** and **6h into the future** (48h for :ref:`aggregate-bonded` transactions). Transactions announced with a deadline outside this window will be rejected with an **invalid deadline** error.
+
+Deadlines are given in **milliseconds since the creation of the nemesis block**.
+
+The moment when the :ref:`nemesis block <block-creation>` was created can be found in the ``network.epochAdjustment`` property of the `/network/properties <https://docs.symbolplatform.com/symbol-openapi/v1.0.1/#operation/getNetworkProperties>`__ endpoint. This is the **number of seconds** elapsed since the `UNIX epoch <https://en.wikipedia.org/wiki/Unix_time>`__ and **it is always 1615853185 for MAINNET**.
+
+In other words, you need to substract the **epoch adjustment** from a Unix time to obtain a deadline. Therefore, a **deadline 2h into the future**, which is the default deadline provided by the SDK, can be calculated as:
+
+.. code-block:: typescript
+
+   currentTime = now(); // Seconds since the UNIX epoch
+   deadline = (currentTime + 7200 - epochAdjustment) * 1000;
+
+This deadline can now be used when building the transaction, and it will expire 2h from now (7200 seconds).
+
+Transaction fee
+===============
+
+The effective :doc:`Fee <../../concepts/fees>` a transaction must pay to be announced is the **transaction size** (in bytes) times a **fee multiplier** chosen by the node that confirms the transaction.
+
+Since this multiplier is unknown when making the announcement, **transactions** define the **maximum fee** they are willing to pay.
+
+Moreover, **nodes** can define a **minimum fee** below which transactions are just ignored.
+
+As a consequence, choosing the right maximum fee for a transaction is very important: Too low a number and the transaction will not be confirmed by any node and will eventually expire. Too high maximum fees, though, will incur in unnecessary expenses.
+
+To help choose the right amount, the `/network/fees/transaction <https://docs.symbolplatform.com/symbol-openapi/v1.0.1/#operation/getTransactionFees>`__ endpoint provides some statistics regarding the effective fees paid by the last 60 blocks. The returned data is:
+
+.. csv-table::
+   :header: "Property", "Meaning"
+   :widths: 18 82
+   :delim: ;
+
+   Highest; Biggest fee multiplier used in the last 60 blocks.
+   Median; The `median value <https://en.wikipedia.org/wiki/Median>`__ of the fee multipliers used in the last 60 blocks.
```

Comment:
Zero median fees calculated will be replaced by the defaultDynamicFeeMultiplier, this has been included in the latest rest (v2.3.6)

Reply:
Added a note!